src/components/Reactions/ReportActionItemEmojiReactions.js

```diff
@@ -132,24 +138,29 @@ function ReportActionItemEmojiReactions(props) {
                             key={reaction.reactionEmojiName}
                         >
                             <View>
-                                <EmojiReactionBubble
-                                    ref={(ref) => (popoverReactionListAnchors.current[reaction.reactionEmojiName] = ref)}
-                                    count={reaction.reactionCount}
-                                    emojiCodes={reaction.emojiCodes}
-                                    onPress={reaction.onPress}
-                                    reactionUsers={reaction.reactionUsers}
-                                    hasUserReacted={reaction.hasUserReacted}
-                                    onReactionListOpen={reaction.onReactionListOpen}
-                                    shouldBlockReactions={props.shouldBlockReactions}
-                                />
+                                <OfflineWithFeedback
+                                    pendingAction={reaction.hasPendingAction}
+                                    shouldDisableOpacity={Boolean(reportAction.pendingAction)}
```

Comment:
```suggestion
                                    shouldDisableOpacity=Boolean(lodashGet(reportAction, 'pendingAction'))
```
`pendingAction` isn't guaranteed to exist